minio/api.py

```diff
@@ -894,19 +913,25 @@ def list_objects_v2(self, bucket_name, prefix=None, recursive=False):
             for obj in objects:
                 yield obj
 
-    def stat_object(self, bucket_name, object_name):
+    def stat_object(self, bucket_name, object_name, sse=None):
         """
         Check if an object exists.
 
         :param bucket_name: Bucket of object.
         :param object_name: Name of object
         :return: Object metadata if object exists
         """
+
+        headers = {}
+
+        if sse:
```

Comment:
The only valid sse type here is `SSE-C`, you should check if it is of that type, if it is passed